example/python/prepare.sh

```diff
@@ -4,7 +4,7 @@ cd $(dirname $0)
 
 # generate iroha lib
 cmake -DSWIG_PYTHON=ON ../..
-make _iroha
+make -j"$(getconf _NPROCESSORS_ONLN)" irohapy
```

Comment:
better remove ```-j```, maybe linux users do not have such command

Reply:
As far as I googled, it should be. Could you check it on your Linux laptop?